e2etest/zt_newe2e_file_test.go

```diff
@@ -546,3 +546,53 @@ func (s *FileTestSuite) Scenario_CopyTrailingDotUnsafeDestination(svm *ScenarioV
 		Body: body,
 	}, false)
 }
+
+// Test empty files are not uploaded when File share quota is hit
+func (s *FileTestSuite) Scenario_UploadFilesWithQuota(svm *ScenarioVariationManager) {
+	quotaMB := int32(10) // 10 MB quota
+	shareName := "testsharewithquota"
```

Comment:
consistent share names can cause problems when testing with static accts, using random names is better; using nil in the ContainerName field automatically creates a random name.